AVL_tree/AVL_tree/Class1.cs

```diff
@@ -0,0 +1,274 @@
+﻿using System;
+
+namespace AVL_tree
+{
+    class Program
```

Comment:
Lets this logic will be in another project. Class library must contain only dictionary, that working with AVL tree rules